gini/gini-vision-lib-android
# Event tracking

Event tracking as described in our [design document](https://docs.google.com/document/d/1iQQD2_XT3U6WuZoFIkd9-J5wyAFxW-mCxWRj6yMYxW8/edit#heading=h.t6baawbhdx7a).

### How to test
Run the screen api example app and look for info logs in logcat for each event. See the `GVLEventTracker` in the screen api example app's `MainActivity` for the log messages.

### Ticket 
[PIA-485](https://ginis.atlassian.net/browse/PIA-485)
